src/Form/matching-config/matching-config-source.js

```diff
@@ -268,7 +268,7 @@ const matchingConfiguration = {
                     forceUnknown: 'captcha|mfa|2fa|two factor|otp|pin'
                 },
                 newPassword: {
-                    match: 'new|re.?(enter|type)|repeat|update|reset\\b'
+                    match: 'new|re.?(enter|type)|repeat|update\\b'
```

Comment:
This is the only relevant business logic change, other changes refactors. And there are tests of course!